docs/solutions/sql_server_always_on_availability_groups/configuration.tf

```diff
@@ -0,0 +1,69 @@
+# configuration.tf - General configuration of SQL Always On deployment
+# Set variables here to meet your needs within your particular environment
+# Check both this file and sql.tf!
+
+# image_id - OCID of the Windows image to use.  This image is the basis for 
+# all instances deployed for  SQL Always On
+# BE SURE TO UPDATE WITH MOST CURRENT LIST OF WINDOWS 2012 or 2016 IMAGES.
+variable "image_id" {
+  type = "map"
+  default = {
```

Comment:
Add comment that this is Windows-Server-2012-R2-Standard-Edition-VM-2018.02.01-0

Reply:
Done.